go.mod

```diff
@@ -4,15 +4,15 @@ go 1.20
 
 require (
 	github.com/ipfs/go-ds-badger2 v0.1.3
-	github.com/ipfs/go-log v1.0.5
```

Comment:
`go-libp2p` is using the `v2` logger. I believe not using `v2` was causing compilation issues, but if you want me to go back and check, I can. Interestingly, `go-libp2p` is not using `badger2` like we are above:
https://github.com/libp2p/go-libp2p/blob/master/go.mod#L18